cumulusci/tasks/salesforce/insert_record.py

```diff
@@ -0,0 +1,24 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.utils import process_list_of_pairs_dict_arg
+
+
+class InsertRecord(BaseSalesforceApiTask):
+    task_docs = """
+        For example:
+
+        cci task run insert_record --org dev -o object PermissionSet -o values Name:HardDelete,PermissionsBulkApiHardDelete:true
+    """
+
+    task_options = {
+        "object": {"description": "An SObject type to insert", "required": True},
+        "values": {
+            "description": "Field names and values in the format 'aa:bb,cc:dd'",
+            "required": True,
+        },
+    }
+
+    def _run_task(self):
+        object_handler = getattr(self.sf, self.options["object"])
+        values = process_list_of_pairs_dict_arg(self.options["values"])
```

Comment:
This should be done in _init_options so that errors parsing the arg are caught before the flow starts running.

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1545/commits/4e4a1a37380e77f0e87f04afbcc7760372093f78#diff-dfd20195f14b74746c715a851a4d98b8R21